### ISSUE TITLE
Angular 5 Module build failed

### ISSUE THREAD

**Issue by Stradivario** [HNeukermans/ng2-signalr#104]
Angular 5 throws error 

ERROR in ./node_modules/ng2-signalr/index.ts
Module build failed: Error: /home/kristiyantachev/Desktop/repos/private/112_operator/node_modules/ng2-signalr/index.ts is not part of the compilation output. Please check the other error messages for details.
    at AngularCompilerPlugin.getCompiledFile (/home/kristiyantachev/Desktop/repos/private/112_operator/node_modules/@ngtools/webpack/src/angular_compiler_plugin.js:648:23)
    at plugin.done.then (/home/kristiyantachev/Desktop/repos/private/112_operator/node_modules/@ngtools/webpack/src/loader.js:467:39)
    at process._tickCallback (internal/process/next_tick.js:109:7)
 @ ./src/app/core/signalr/signalr.module.ts 9:0-44 10:0-51
 @ ./src/app/core/core.module.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi webpack-dev-server/client?http://0.0.0.0:0 ./src/main.ts

webpack: Failed to compile.

**Comment by thduttonuk** [comment]
This is fixed, just waiting on an NPM publish @HNeukermans

**Comment by AndonyNS** [comment]
Hi! Has this been published? Thanks

**Comment by thduttonuk** [comment]
@AndonyNS a publish has just gone out
